src/styles/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(207, 207, 207);
+}
+
+p {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.like-button {
+  cursor: pointer;
+}
+
+/* .card-img-top {
+  width: 100%;
+  height: 30vw;
+  object-fit: cover;
+} */
```

Comment:
### **Optional**

- [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain

Reply:
Thank you very much for this review. I will get it sorted out